src/components/TaskCardButtonsContainer/TaskCardButtonsContainer.jsx

```diff
@@ -0,0 +1,37 @@
+import PropTypes from "prop-types";
+import { getDaysToCompleteTask } from "src/utils/compareDate";
+import DeleteButton from "src/components/Buttons/DeleteButton/DeleteButton";
+import DoneButton from "src/components/Buttons/DoneButton/DoneButton";
+
+function TaskCardButtonsContainer({
+  isTaskDone,
+  createdAt,
+  taskDoneAt,
+  handleDeleteTask,
+  handleTaskDone,
+}) {
+  const numberOfDays = getDaysToCompleteTask(createdAt, taskDoneAt);
```

Comment:
```suggestion
  const numberOfDays = getDaysToCompleteTask(createdAt, completedAt);
```

Reply:
@iud-wtag as the task description in trello uses the term `done`, I am using `done` instead of `complete` for consistency.